metricbeat/helper/http.go

```diff
@@ -0,0 +1,85 @@
+package helper
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/elastic/beats/metricbeat/mb"
+)
+
+type Http struct {
```

Comment:
:-) @tsg and me had the same discussion, but came to the conclusion HTTP will look ugly :-) What would you call the private variable in this case? http?